openslides_backend/action/group/create.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any, Dict
+
+from ...models.models import Group
+from ..default_schema import DefaultSchema
+from ..generics import CreateAction
+from ..register import register_action
+
+
+@register_action("group.create")
+class GroupCreate(CreateAction):
+    """
+    Action to create a group.
+    """
+
+    model = Group()
+    schema = DefaultSchema(Group()).get_create_schema(
+        required_properties=["name", "meeting_id"],
+        additional_optional_fields={
+            "permissions": {"type": "array", "items": {"type": "string"}}
+        },
+    )
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Set default permissions to list().
+        """
+        if instance.get("permissions") is None:
+            instance["permissions"] = []
+        return instance
```

Comment:
This should not be necessary, the "default default" for array fields is already `[]`. Please remove and test if it still works. Independent from that, please implement defaults that differ from the "default defaults" in the models.yml and not in the actions themselves to keep the handling consistent.

Reply:
Okay, cutted and tested, okay.